beezee/assetlist.json

```diff
@@ -609,6 +609,40 @@
         "twitter": "https://x.com/gge_crypto"
       },
       "type_asset": "sdk.coin"
+    },
+    {
+      "description": "$CryptoTrtl is fun, join us on Telegram.",
+      "denom_units": [
+        {
+          "denom": "factory/bze1972aqfzdg29ugjln74edx0xvcg4ehvysjptk77/CTL",
+          "exponent": 0
+        },
+        {
+          "denom": "CTL",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/bze1972aqfzdg29ugjln74edx0xvcg4ehvysjptk77/CTL",
+      "name": "CryptoTrtl",
+      "display": "CTL",
+      "symbol": "CTL",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/beezee/images/ctl.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/beezee/images/ctl.png",
+          "theme": {
+            "primary_color_hex": "#FEFEFE"
+          }
+        }
+      ],
+      "socials": {
+        "website": "https://app.getbze.com/token?denom=factory%2Fbze1972aqfzdg29ugjln74edx0xvcg4ehvysjptk77%2FCTL",
```

Comment:
Please remove this. Let's keep the website property for a website that showcases the project--not just a pre-selected swap tool for buying

Reply:
Hey, Jeremy! I thought it's mandatory. I removed it entirely or should I leave it blank ?

Reply:
Thank you. It's best to not include empty properties, either.